src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
+
+  sourceString
+    .split(';')
+    .map((declaration) => declaration.trim())
+    .filter((declaration) => declaration)
+    .forEach((declaration) => {
+      const [property, value] = declaration
+        .split(':')
+        .map((part) => part.trim());
```

Comment:
The current implementation splits the declaration by the first colon found. This is correct according to the task requirements, as handling multiple colons is not required. However, be aware that if there are multiple colons, this implementation will not handle them correctly, which is acceptable for this task.